src/plugins/embeddable/public/react_embeddable_system/react_embeddable_renderer.tsx

```diff
@@ -14,8 +14,7 @@ import {
 import { PresentationPanel, PresentationPanelProps } from '@kbn/presentation-panel-plugin/public';
 import { ComparatorDefinition, StateComparators } from '@kbn/presentation-publishing';
 import React, { useEffect, useImperativeHandle, useMemo, useRef } from 'react';
-import { combineLatest } from 'rxjs';
-import { debounceTime, skip } from 'rxjs/operators';
+import { combineLatest, debounceTime, skip } from 'rxjs';
```

Comment:
Maybe I rebased main at a strange time. Otherwise, I have no idea why this came up as a lint issue and caused a failure in my PR :/

Reply:
That's the reason https://github.com/elastic/kibana/pull/179553 😊

Reply:
Thanks for fixing this @tsullivan!